tools/recipes/rt-app.recipe

```diff
@@ -9,11 +9,21 @@ build_jsonc() {
     make
 }
 
+build_numactl() {
```

Comment:
The recipe looks good to me, but I'll need to get clearance from our legal department before merging the PR since we distribute prebuilt binaries of rtapp in LISA.

If you want, you can split the PR into two parts:
* build recipe update for numactl
* everything else

This way we can merge the Python side faster and avoid potential merge conflicts arising from stale PRs.

Reply:
I will remove build recipe update from this PR 
and will create a separate PR for new rt-app and numactl.